xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -828,6 +828,30 @@ When submitting a certificate signing request via a web browser or other graphic
         <exception cref="T:System.ArgumentNullException">
           <paramref name="signatureGenerator" /> is <see langword="null" />.</exception>
         <exception cref="T:System.Security.Cryptography.CryptographicException">A cryptographic error occurs while creating the signing request.</exception>
+        <exception cref="T:System.InvalidOperationException">
+          <see cref="P:System.Security.Cryptography.X509Certificates.CertificateRequest.OtherRequestAttributes" /> contains a <see langword="null" /> value.
```

Comment:
These properties exist too, so I'm not understanding the warning:

[src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/CertificateRequest.cs#L45](https://github.com/dotnet/runtime/blob/761f7740588da647dedde2a38194117da3890b09/src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/CertificateRequest.cs#L45)

Reply:
Ah, it does not exist in Preview7. I'm removing this exception.
https://github.com/dotnet/runtime/blob/release/7.0-preview7/src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/CertificateRequest.cs